src/lib.rs

```diff
@@ -625,13 +980,23 @@ impl HarnessApi {
     /// gets to a block via one of `create_block*()` methods.
     pub fn post_private<T, D>(&self, kind: ApiKind, endpoint: &str, transaction: &T) -> D
     where
-        T: Transaction + Serialize,
+        T: Serialize,
         for<'de> D: Deserialize<'de>,
     {
-        HarnessApi::post_internal(
+        TestKitApi::post_internal(
             &self.private_mount,
             &format!("{}/{}", kind.into_prefix(), endpoint),
             transaction,
         )
     }
 }
+
+#[test]
+fn test_create_block_heights() {
+    let mut testkit = TestKitBuilder::validator().create();
+    assert_eq!(Height(1), testkit.current_height());
+    testkit.create_block();
+    assert_eq!(Height(2), testkit.current_height());
+    testkit.create_blocks_until(Height(6));
+    assert_eq!(Height(7), testkit.current_height());
```

Comment:
Hm, I'd think that:
- `TestKit.current_height()` would return the height of the blockchain (i.e., the number of blocks in it). This seems to hold in the current implementation.
- `create_blocks_until(h)` should create blocks until the blockchain reaches height `h`; i.e., after the operation, the blockchain height would be exactly equal to `h`. This seems not to hold in the current implementation; instead, after the operation, the blockchain height is equal to `h + 1`.

That is, the following test would hold:

```rust
let h = testkit.current_height();
testkit.create_blocks_until(h);
assert_eq!(h, testkit.current_height());
```

Reply:
I implemented `last_block_height` for height of latest committed block.